### PR TITLE
added Aryan to extended team

### PR DESCRIPTION
**Pull request by aryangupta701**

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: site/content/docs/team/aryan.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Aryan is an enthusiastic advocate for open-source development, actively contributing to multiple projects in the open-source community. With a profound passion for software development, he possesses a strong aptitude for creative problem-solving. 
+
+His expertise spans a wide range of programming languages, including Java, JavaScript, TypeScript, C++, and Python.
+
+Aryan's proficiency as a full-stack developer, particularly in the MERN stack, reflects his commitment to continuous learning and staying at the forefront of technology.
+
+His inquisitiveness extends to the realm of cybersecurity, where he thrives on the challenge of identifying and mitigating online vulnerabilities.
+
+He was the Google Summer of Code 2023 student contributor for the [Browser Recorder](/blog/2023-09-11-browser-recorder/index.md) feature for ZAP.
```
Comment (thc202) [inline site/content/docs/team/aryan.md:21]: Remove the `index.md`.

Comment (aryangupta701) [reply site/content/docs/team/aryan.md:21]: okay

Comment (thc202) [reply site/content/docs/team/aryan.md]: The last slash should be kept though.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you! (This will have to wait for the blog to be ready.)

**Review by kingthorin — APPROVED** [review]

File: site/content/docs/team/aryan.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+His expertise spans a wide range of programming languages, including Java, JavaScript, TypeScript, C++, and Python.
+
+Aryan's proficiency as a full-stack developer, particularly in the MERN stack, reflects his commitment to continuous learning and staying at the forefront of technology.
+
+His inquisitiveness extends to the realm of cybersecurity, where he thrives on the challenge of identifying and mitigating online vulnerabilities.
+
+He was the Google Summer of Code 2023 student contributor for the [Browser Recorder](/blog/2023-09-11-browser-recorder/) feature for ZAP. 
+
+Contact Him Here: [GitHub](https://www.github.com/aryangupta701) | [LinkedIn](https://www.linkedin.com/in/aryan-gupta-78273a1b6/) | [Twitter](https://twitter.com/aryangupta_701)
```
Comment (psiinon) [inline site/content/docs/team/aryan.md:23]: Dont really need these as they are already in the left panel?
Otherwise all good 😁

Comment (aryangupta701) [reply site/content/docs/team/aryan.md:23]: oh yes right

File: site/content/docs/team/aryan.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Aryan is an enthusiastic advocate for open-source development, actively contributing to multiple projects in the open-source community. With a profound passion for software development, he possesses a strong aptitude for creative problem-solving. 
+
+His expertise spans a wide range of programming languages, including Java, JavaScript, TypeScript, C++, and Python.
+
+Aryan's proficiency as a full-stack developer, particularly in the MERN stack, reflects his commitment to continuous learning and staying at the forefront of technology.
+
+His inquisitiveness extends to the realm of cybersecurity, where he thrives on the challenge of identifying and mitigating online vulnerabilities.
+
+He was the Google Summer of Code 2023 student contributor for the [Browser Recorder](/blog/2023-09-11-browser-recorder/) feature for ZAP.
```
Comment (psiinon) [inline site/content/docs/team/aryan.md:21]: If you remove the link then we can merge it now. Can then add the link in the same PR as the blog post...

Comment (aryangupta701) [reply site/content/docs/team/aryan.md:21]: okayy.

**Review by psiinon — APPROVED** [review]

**Pull request merged**